tests/vmi_multus_test.go

```diff
@@ -867,10 +868,12 @@ var _ = Describe("[Serial]SRIOV", func() {
 		}
 
 		It("[test_id:3956]should connect to another machine with sriov interface over IPv4", func() {
+			Skip("Skip until https://github.com/kubevirt/kubevirt/issues/3774 fixed")
```

Comment:
since 4001 points now on 3840, this should point on 3840 as well please.
(and also on 4432)

see please
https://github.com/kubevirt/kubevirt/pull/4438#issuecomment-718091502


same for line 876

Reply:
#3774 is the first issue that opened by the user to report flakyness
any other issue that adding more to that is referenced in that issue.
As we discussed it in the main PR page lets continue there, 
I am resolving this comment